website/src/data/components/network/props.js

```diff
@@ -150,6 +150,12 @@ const props = [
         defaultValue: NetworkDefaultProps.nodeBorderColor,
         controlType: 'inheritedColor',
     },
+    {
+        key: 'nodeComponent',
+        group: 'Nodes',
+        type: '(node: Node) => svg',
```

Comment:
It's not correct if we consider canvas, you can add `flavors: ['svg'],` if you remove support for canvas, or adjust the type to use a union otherwise.

Reply:
Similar to the swarmplot implementation I have added a `nodeComponent` for svg flavor and `renderNode` for canvas flavor.

Reply:
Makes sense!